web/src/components/software/PatternSelector.jsx

```diff
@@ -38,36 +35,10 @@ import { _ } from "~/i18n";
  * @property {string} description long description of the pattern
  * @property {string} order display order (string!)
  * @property {string} icon icon name (not path or file name!)
- * @property {number|undefined} selected who selected the pattern, undefined
+ * @property {number} selected who selected the pattern, undefined
```

Comment:
NP: using a different style here. I know it is following the previous style, just adding a note. Also, looks like it is not undefined anymore.